src/components/BorrowerProfile/MoreAboutLoan.vue

```diff
@@ -40,22 +41,35 @@
 		<div
 			v-if="!partnerName && !loading"
 		>
-			<h3>
-				Business description
-			</h3>
-			<p
-				:key="`businessDescription-${index}`"
-				v-html="this.businessDescription"
-			>
-			</p>
-			<h3>
-				What is the purpose of this loan?
-			</h3>
-			<p
-				:key="`purpose-${index}`"
-				v-html="this.purpose"
-			>
-			</p>
+			<div class="tw-prose tw-mb-2">
```

Comment:
I had to segment the application of tw-prose so that it wasn't applied on the new component.